content/en/docs/collector/howto-custom-collector.md

```diff
@@ -0,0 +1,112 @@
+# HOW-TO: Build your own Collector instance
+
+If you are planning to build and debug your components, you are going to need your own Collector instance. That will allow you to launch and debug your Collector components directly within your favorite Golang IDE.
+
+The other really cool aspect of approaching the component development this way is that you can use all the cool debugging features from your IDE (stack traces are great teachers!) to understand how the Collector itself interacts with your component code.
+
+In order to make it easy for developers to create their own custom builds of the Collector, the OpenTelemetry Community has developed a tool called `OpenTelemetry Collector Builder`.
+
+The `builder` is meant to help developers quickly assemble and build their own collector's distributions, based on a given configuration file.
```

Comment:
check that you consistently write "collector" or "Collector", from a few other pages it looks like that it is "Collector"

Reply:
Hi @svrnm, fixed on [7d3d01](https://github.com/open-telemetry/opentelemetry.io/pull/1277/commits/7d3d01bd356ff8c6ae72ffd52cc476f69108f7ca)